datasets/pk/proscribed_persons/crawler.py

```diff
@@ -0,0 +1,50 @@
+from zavod import Context
+from zavod import helpers as h
+
+
+def crawl_person(context: Context, row: dict):
+    person_name = row.pop("name")
+    father_name = row.pop("fatherName")
+    cnic = row.pop("cnic")
+    province = row.pop("province")
+    district = row.pop("district")
+
+    entity = context.make("Person")
+    entity.id = context.make_slug(
+        cnic or f"{person_name} - {district}, {province}", prefix="pk-cnic"
+    )
```

Comment:
nice approach, but let's not override the prefix when falling back to person_name and location. Let it default to the prefix in the metadata in that case.

The idea with pk-cnic is that it then becomes a very readable national ID schema so we don't want to mix in names and things in there.